gips/data/landsat/landsat.py

```diff
@@ -1001,20 +1016,43 @@ def process(self, products=None, overwrite=False, **kwargs):
 
                     elif val[0] == 'cloudmask':
                         qaimg = self._readqa()
-                        npqa = qaimg.Read() # transmogrify into numpy array
+                        npqa = qaimg.Read()  # read image file into numpy array
                         # https://landsat.usgs.gov/collectionqualityband
                         # cloudmaskmask = (cloud and (cc_med or cc_high)) or csc_med or csc_high
                         # cloud iff bit 4
                         # (cc_med or cc_high) iff bit 6
                         # (csc_med or csc_high) iff bit 8
+
                         def get_bit(np_array, i):
                             """Return an array with the ith bit extracted from each cell."""
                             return (np_array >> i) & 0b1
-                        np_cloudmask = get_bit(npqa, 4) & get_bit(npqa, 6) | get_bit(npqa, 8)
-                        imgout = gippy.GeoImage(fname, img, gippy.GDT_Byte, 1) # only one layer
+
+                        np_cloudmask = numpy.logical_not(
+                            get_bit(npqa, 4) &
+                            get_bit(npqa, 6) |
+                            get_bit(npqa, 8)
+                        )
+
+                        # We already have an implicit dependency on scipy (from
+                        # Py6S), but life might be more simple if we just move
+                        # the whole cloudmask extraction into gippy.
+                        from scipy import ndimage
```

Comment:
optional:  Is it worth it to add scipy to gips dependencies so it's installed explicitly?  Also should this import go at the top of the file?